public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -553,21 +554,21 @@ block dart-type-exception-example
 a(id="one-time-initialization")
 :marked
   ### One-time string initialization
-  We *should* omit the brackets when all of the following are true:
+  The brackets *should* be omited when the following conditions are true:
   * The target property accepts a string value.
   * The string is a fixed value that we can bake into the template.
   * This initial value never changes.
 
-  We routinely initialize attributes this way in standard HTML, and it works
+  This initialization is routinely used in standard HTML, and it works
   just as well for directive and component property initialization.
   The following example initializes the `prefix` property of the `HeroDetailComponent` to a fixed string,
-  not a template expression. Angular sets it and forgets about it.
+  not a template expression. In this case, Angular sets it and forgets about it.
 +makeExample('template-syntax/ts/app/app.component.html', 'property-binding-7')(format=".")
 :marked
   The `[hero]` binding, on the other hand, remains a live binding to the component's `currentHero` property.
 
   ### Property binding or interpolation?
-  We often have a choice between interpolation and property binding. 
+  There often exists a choice between interpolation and property binding.
```

Comment:
You're right to avoid "we", but I also usually avoid "there" in technical writing... it's a content-free word that's a clue that the writing could be clearer.

Once we rewrite this whole doc to use "you" instead of "we", rewriting this sentence will be easier...

Our task here is more difficult because the text isn't clear. I don't think the point is that a choice exists, but that either choice or both might be valid. In this case, I'd follow up with the author to make sure we're communicating the right thing. Maybe this text and example aren't even needed.